playbooks/fedora-30/test-with-podman-stable.yaml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: ci-node-30
+  tasks:
+    - name: Build toolbox
+      command: meson builddir
+      args:
+        chdir: '{{ zuul.project.src_dir }}'
+
+    - name: Install toolbox
+      become: yes
+      command: ninja -C builddir install
+      args:
+        chdir: '{{ zuul.project.src_dir }}'
+
+    - name: Run system tests
+      command: bats ./test/system
```

Comment:
Nitpick: we should swap the order of commits so that `./test/system` is in place before we start referring to it.